_data/simple-icons.json

```diff
@@ -8557,6 +8557,12 @@
             "source": "https://www.ros.org/press-kit/",
             "guidelines": "https://www.ros.org/press-kit/"
         },
+        {
+            "title": "Rossmann",
+            "hex": "C3002D",
+            "source": "https://www.rossmann.de",
+            "guidelines": "https://www.rossmann.de/unternehmen/presse/media-downloads/logos.html"
```

Comment:
Icon is a perfect match, but this page does not seem to exist. It just redirects to https://unternehmen.rossmann.de/presse/mediathek.html which is not a guidelines page.